packages/next/src/build/webpack-config.ts

```diff
@@ -213,6 +213,9 @@ function createRSCAliases(
     'react-server-dom-webpack/client.edge$': `next/dist/compiled/react-server-dom-webpack${bundledReactChannel}/client.edge`,
     'react-server-dom-webpack/server.edge$': `next/dist/compiled/react-server-dom-webpack${bundledReactChannel}/server.edge`,
     'react-server-dom-webpack/server.node$': `next/dist/compiled/react-server-dom-webpack${bundledReactChannel}/server.node`,
+    // optimisations to ignore the legacy build of react-dom/server
+    './cjs/react-dom-server-legacy.browser.production.min.js': `next/dist/build/noop-react-dom-server-legacy`,
```

Comment:
Can we specify the package name and skip the extension to merge into one

```js
'react-dom/cjs/react-dom-server-legacy.browser': 'next/dist/build/noop-react-dom-server-legacy'
```

Reply:
the alias is relative AFAIK so I can't do that